### ISSUE TITLE
Delete this repository

### ISSUE THREAD

**Issue by danfowler** [dataprotocols/data-packages#17]
@rgrp OK to delete? Can't tell what the benefit is of keeping around.

**Comment by rgrp** [comment]
@danfowler yes you can delete (purpose was to split out one repo for each spec - but we decided against this).
